GAE/src/org/akvo/flow/api/app/DataPointUtil.java

```diff
@@ -126,20 +125,37 @@ private SurveyedLocaleDto createSurveyedLocaleDto(Long surveyGroupId, QuestionDa
      * Fetches SurveyInstances using the surveyedLocalesIds and puts them in a map:
      * key: SurveyedLocalesId, value: list of SurveyInstances
      */
-    private Map<Long, List<SurveyInstance>> getSurveyInstances(List<Long> surveyedLocalesIds) {
+    public Map<Long, List<SurveyInstance>> getSurveyInstances(List<Long> surveyedLocalesIds, Long surveyId, Device device) {
```

Comment:
I see this method has been made public for tests: https://www.artima.com/suiterunner/private.html